tribute.css

```diff
@@ -0,0 +1,34 @@
+.tribute-container {
```

Comment:
Hi. Why this file was added. It seems like a (formatted) copy of `dist/tribute.css`. Just curious. Thanks.